src/site/xdoc/checks/coding/textblockgooglestyleformatting.xml

```diff
@@ -0,0 +1,268 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<document xmlns="http://maven.apache.org/XDOC/2.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/XDOC/2.0 https://maven.apache.org/xsd/xdoc-2.0.xsd">
+  <head>
+    <title>TextBlockGoogleStyleFormatting</title>
+  </head>
+  <body>
+    <section xmlns="http://maven.apache.org/XDOC/2.0"
+             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="TextBlockGoogleStyleFormatting">
+      <p>Since Checkstyle 12.2.0</p>
+      <subsection name="Description" id="Description">
+        <div>
+          Checks correct format of
+          <a href="https://docs.oracle.com/en/java/javase/17/text-blocks/index.html">Java Text Blocks</a>
+          as specified in
+          <a href="https://google.github.io/styleguide/javaguide.html#s4.8.9-text-blocks">
+          Google Java Style Guide</a>.
+        </div>
+
+        <p>
+          This Check has two functions: First is to make sure that both opening and closing quotes
+          (<code>"""</code>) of text block are on new lines i.e no other item should precede the quotes and
+          second is to ensure opening and closing quotes are vertically aligned.
+          Note: Closing quotes can be followed by further code on the same line.
```

Comment:
this is very hard to read exlanation block.
can we format it a bit to looks like (and some wording changes):
```
There are two requirements: 
- both opening and closing quotes (""") of text block are on new lines i.e no other item should precede the quotes;
- opening and closing quotes are vertically aligned. 
 
Note: Closing quotes can be followed by further code on the same line. 
```